lib/puppet/provider/mongodb_user/mongodb.rb

```diff
@@ -60,9 +60,11 @@ def self.prefetch(resources)
   def create
     if db_ismaster
       if mongo_24?
+        #if mongodb <= 2, the creation of the user involves a plaintext password (hashes are not accepted)
+        fail("Property 'password_plain' must be set for mongoDB 2.4.x.") unless @resource[:password_plain]
```

Comment:
added the parameter password_plain; when creating a user, if mongodb<=2.4 and this parameter is not specified, the creation fails